securedrop/db.py

```diff
@@ -131,7 +131,7 @@ class Submission(Base):
     source = relationship(
         "Source",
         backref=backref("submissions", order_by=id, cascade="delete")
-        )
+    )
```

Comment:
Not a fan of this C-style syntax to be honest. I prefer to fit more content in a page view, so long as it doesn't hurt readability. IMO the right-hand side is not so complicated that putting it into two lines would hurt readability. I do use it [here](https://github.com/freedomofpress/securedrop/pull/1467/files#diff-930c74950e483448f44f7d78910b23a8R155) for example, where the two vars in the brackets would otherwise be flush with the rest of the arguments, making it otherwise harder to tell they were nested in a list.